File: src/org/r3/db/mongo/quiz/quizuseranswercollection/QuizUserAnswerCollectionDTO.java
//
// Understanding Department - 2016 
//
//
// Mongo - TODO: Readme Section.
//




package org.r3.db.mongo.quiz.quizuseranswercollection;

import org.r3.db.MongoDbSchemaInfo;
import org.mongodb.morphia.annotations.Entity;

/*
 * Please Define in MongoDbSchemaInfo: public static final String MongoDbQuizUserAnswerCollectionName = "r3_quizuseranswers";
 */
@Entity(value=MongoDbSchemaInfo.MongoDbQuizUserAnswerCollectionName, noClassnameStored=true)
public class QuizUserAnswerCollectionDTO extends QuizUserAnswerCollectionDTOBase{
	public QuizUserAnswerCollectionDTO()
    {
        super();
    }
}

File: WebContent/org/r3/db/mongo/quiz/quizuseranswercollection/QuizUserAnswerCollectionDTO.js
//
// Understanding Department - 2016 
//
//
// Mongo - TODO: Readme Section.
//


jQuery.r3Utils.namespace("org.r3.db.mongo.quiz.quizuseranswercollection");

org.r3.db.mongo.quiz.quizuseranswercollection.QuizUserAnswerCollectionDTO = Class.extend(
{
	 init: function(aDataBean)
	 {  
       this.dtrowclass   = "grade_quizuseranswercollection";
       this.dtokey       = aDataBean.dtokey   || null;
       this.dtodescr     = aDataBean.dtodescr || null;
       
       this.useranswerid = aDataBean.useranswerid || null;
       this.quizid = aDataBean.quizid || null;
       this.usersessionid = aDataBean.usersessionid || null;
       this.quizuserblocknumber = aDataBean.quizuserblocknumber || null;
       this.blocknumber = aDataBean.blocknumber || null;
       this.questionid = aDataBean.questionid || null;
       this.answerid = aDataBean.answerid || null;
       this.valid = aDataBean.valid || null;
       this.score = aDataBean.score || null;
       this.when = aDataBean.when || null;
       this.timeout = aDataBean.timeout || null;
	 }
       
});

